apps/presence/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:20-alpine
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider using specific Node.js version tag</b></div>
<div id="fix">

Consider using a more specific Node.js version tag instead of `20-alpine`. This could help ensure consistency across different environments and prevent potential issues due to unexpected changes in newer versions.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -1 +1 @@
 -FROM node:20-alpine
 +FROM node:20.5.1-alpine3.18
```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/omkardongre/messenger-api/pull/6#issuecomment-2448418554>#48ef5b</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged